### PR TITLE
Create ci.yml

### PR DESCRIPTION
**Pull request by DenysGonchar**
Migrating from Travis CI to GH Actions

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+         with:
+           fetch-depth: 0
+       - uses: ErlGang/setup-erlang@v1.0.0
+         with:
+           otp-version: ${{ matrix.otp }}
+       - name: Compile project
+         run: ./rebar3 compile
+       - name: run xref check
+         run: ./rebar3 xref
+       - name: run dializer check
```
Comment (chrzaszcz) [inline .github/workflows/ci.yml:29]: minor typo in `dializer`

File: travis/test_docker_image.sh [hunk 2]
```diff
@@ -1,29 +0,0 @@
...
-	       --health-cmd="${PATH_TO_EXEC} status" \
-	       amoc-arsenal-xmpp:latest
-
-./travis/wait_for_healthcheck.sh amoc-arsenal-1
-./travis/wait_for_healthcheck.sh amoc-arsenal-2
-
-docker exec -it amoc-arsenal-1 ${PATH_TO_EXEC} eval "nodes()" | grep amoc-arsenal-2
-docker exec -it amoc-arsenal-2 ${PATH_TO_EXEC} eval "nodes()" | grep amoc-arsenal-1
-
-
```
Comment (chrzaszcz) [inline travis/test_docker_image.sh:29]: very minor: extra newline

**Review by chrzaszcz — APPROVED** [review]
Looks good, the generic N-node test looks strong ;-)
Minor comments, fixing is optional.

File: travis/test_docker_image.sh (hunk 2 above)
Comment (DenysGonchar) [reply travis/test_docker_image.sh:29]: that's an old file, the [new one](https://github.com/esl/amoc-arsenal-xmpp/pull/21/files/a70237e39b1a75ac91a81790302dc5ab8765891b#diff-2fe9639dd0beca7ac77ccd5f3c6806dc4f40faba30c6764e23db74193291ddca) doesn't have that extra new line.

File: .github/workflows/ci.yml (hunk 1 above)
Comment (DenysGonchar) [reply .github/workflows/ci.yml]: fixed

**Pull request merged**